### PR TITLE
FIX-#4414: Add missing f prefix on f-strings found at https://codereview.doctor

### LINKED ISSUE #4414: Missing `f` prefix on f-strings
Some strings looks like they're meant to be f-strings but are missing the `f` prefix meaning variable interpolation won't happen.

https://github.com/modin-project/modin/blob/0ab04e104a60419c464fc1833a13f7f2b92e7983/asv_bench/benchmarks/benchmarks.py#L692
https://github.com/modin-project/modin/blob/0ab04e104a60419c464fc1833a13f7f2b92e7983/modin/core/dataframe/pandas/dataframe/dataframe.py#L326
https://github.com/modin-project/modin/blob/0ab04e104a60419c464fc1833a13f7f2b92e7983/modin/config/envvars.py#L96
https://github.com/modin-project/modin/blob/0ab04e104a60419c464fc1833a13f7f2b92e7983/modin/config/envvars.py#L111

I found this issue automatically. I'm a bot. Beep Boop 🦊. See other issues I found in your repo [here](https://codereview.doctor/modin-project/modin)

### PR DESCRIPTION
**Pull request by code-review-doctor**
Fixes #4414

### CODE REVIEW

**Review by devin-petersohn — COMMENTED** [review]
Hi @code-review-doctor, thanks for the PR!

We use commit messages in our release notes, and have some standard around how they are formatted. Please take a look if you can: https://modin.readthedocs.io/en/stable/development/contributing.html#commit-message-formatting

Commit messages also need to be signed. Let me know if you have any questions!

**Review by anmyachev — APPROVED** [review]

**Comment by anmyachev** [comment]
Thanks @code-review-doctor! I've made some minor changes that are required in order for your pull request to be accepted, I hope you don't mind.

@modin-project/modin-core ready to merge.

**Review by dchigarev — DISMISSED** [review]

**Review by dchigarev — APPROVED** [review]

File: docs/release_notes/release_notes-0.15.0.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ Key Features and Updates
   * FIX-#4373: Fix invalid file path when trying `read_csv_glob` with `usecols` parameter (#4405)
   * FIX-#4394: Fix issue with multiindex metadata desync (#4395)
   * FIX-#4425: Add parameters to groupby pct_change (#4429)
+  * FIX-#4414: Fix issue probably-meant-fstring found at https://codereview.doctor (#4415)
```
[the remark this answers is not in the document]
Comment (anmyachev) [reply docs/release_notes/release_notes-0.15.0.rst]: @dchigarev let's leave a mention of the tool that helped us.

[the remark this answers is not in the document]
Comment (dchigarev) [reply docs/release_notes/release_notes-0.15.0.rst]: no problem, but then we need to leave it in the PR's title as well.

I just heard a rule somewhere, that the PR's title and the release note have to be identical

[the remark this answers is not in the document]
Comment (dchigarev) [reply docs/release_notes/release_notes-0.15.0.rst]: @anmyachev could you please rename the PR and release note the way you like, but to be identical?

[the remark this answers is not in the document]
Comment (anmyachev) [reply docs/release_notes/release_notes-0.15.0.rst]: @dchigarev  done

**Review by vnlitvinov — COMMENTED** [review]
Changes look good, but don't we need the original author to sign off their commits (instead of Modin contributors signing them)?

**Comment by anmyachev** [comment]
> Changes look good, but don't we need the original author to sign off their commits (instead of Modin contributors signing them)?

@vnlitvinov In situations where the original author can't do this, should we somehow upload pull requests?

**Comment by anmyachev** [comment]
@modin-project/modin-core the ball is on your side

**Pull request merged**